api/src/certification/session-management/infrastructure/repositories/assessment-result-jury-comment-repository.js

```diff
@@ -15,4 +15,8 @@ const getLatestAssessmentResult = async function ({ certificationCourseId }) {
   return new AssessmentResultJuryComment(result);
 };
 
-export { getLatestAssessmentResult };
+const save = async function ({ id, juryId, commentByJury }) {
```

Comment:
Pourquoi ne pas nommer `update` (comme présent dans d'autres repo) plutôt que `save` 

Je trouve ça moins explicite pour le coup.

```suggestion
const update = async function ({ id, juryId, commentByJury }) {
```